src/sentry/api/endpoints/organization_eventid.py

```diff
@@ -43,19 +43,19 @@ def get(self, request, organization, event_id):
         if len(event_id) != 32:
             return Response({"detail": "Event ID must be 32 characters."}, status=400)
 
-        # Limit to 100req/s
+        # Limit to 1 req/s
         if ratelimiter.is_limited(
             u"api:event-id-lookup:{}".format(
                 md5_text(
                     request.user.id if request.user and request.user.is_authenticated() else ""
```

Comment:
Hmm, I'm not sure what case a user could be not authenticated here, but we do check for it. This will drop all unauthenticated users into the same bucket. I guess maybe this is a thing for some types of access tokens that aren't associated with a user?

Reply:
I'll have to look into what happens before we hit this point, but if I can confirm a user could not be unauthenticated here then I can modify this line.